### PR TITLE
blendid5: strip tasks/update dependencies

### PR DESCRIPTION
**Pull request by benjtinsley**
in blendid5.0, we are moving toward a much simpler tool with better documentation, instead of continuing to build a robust tool that does everything for you. we find that the more robust the tool, the more difficult it is to maintain in the long term, and the less likely it will be useful, up to date, or bug free.

the first step of accomplishing this goal is to strip out all non-default init tasks, as well as the extras directory that those pull from. any necessary files within `extras/` are now moved to `root/`

this PR also updates any upgradable packages and removes any that are unneeded.

### CODE REVIEW

File: gulpfile.js/tasks/rev/rev-css.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var gulp        = require('gulp')
 var rev         = require('gulp-rev')
-var revNapkin   = require('gulp-rev-napkin')
+var revDel      = require('gulp-rev-delete-original')
```
Comment (gregkohn) [inline gulpfile.js/tasks/rev/rev-css.js]: Particular reason for the swap?

**Comment by olets** [comment]
🎖 for the word obviated

File: gulpfile.js/tasks/init.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
-var gulp = require('gulp')
-var log = require('fancy-log')
-var colors = require('ansi-colors')
+var gulp        = require('gulp')
+var log         = require('fancy-log')
+var colors      = require('ansi-colors')
 var projectPath = require('../lib/projectPath')
-var merge = require('merge-stream')
+var merge       = require('merge-stream')
```
Comment (davist11) [inline gulpfile.js/tasks/init.js]: I'd argue against doing this sort of alignment. Adds unnecessary noise to PRs just for spacing changes.

**Comment by olets** [comment]
this removes the Blendid logo, which appears at the top of `./README.md`. One way to keep it in the readme while removing it from the repo would be to use a rawgit production url

File: gulpfile.js/tasks/rev/rev-css.js (hunk 1 above)
Comment (benjtinsley) [reply gulpfile.js/tasks/rev/rev-css.js]: napkin hasnt been updated in 3 years and delete-original has been updated in the last year. napkin was still reliant on gulp-util that was throwing errors, so i didn't really think the maintainer was going to push up anything anytime soon to resolve this

File: package.json [hunk 3]
```diff
@@ -68,7 +64,9 @@
     "webpack-hot-middleware": "^2.18.2"
   },
   "devDependencies": {
+    "babel-preset-env": "^1.6.1",
     "chai": "^3.5.0",
+    "gulp-rev-delete-original": "^0.2.3",
```
[the remark this answers is not in the document]
Comment (benjtinsley) [reply package.json]: ah yes you are totally right. i have to remind myself to work on blendid as an NPM package, not as a complete product, so a lot of the dependencies must be there by default.

File: gulpfile.js/tasks/init.js (hunk 2 above)
Comment (benjtinsley) [reply gulpfile.js/tasks/init.js]: ah good to know. in the future i won't include these kinds of changes in PRs

**Comment by benjtinsley** [comment]
@olets good catch. i think i'd like to get the blendid logo updated to show it's 5.0 (and remove the gulp-starter bit) just so people coming to the repo see an immediate change.

**Pull request merged**